deploy/hooks.js

```diff
@@ -0,0 +1,118 @@
+const mime = require("mime-types");
```

Comment:
Heavily copied from the monorepo. This copies every file to s3 and then removes ones that are no longer in the repo